packages/alfa-dom/src/jsx.ts

```diff
@@ -16,6 +17,7 @@ export function jsx<N extends string = string>(
 ): Element<N> {
   const attributes: Record<string, string | boolean> = {};
   const style: Record<string, string> = {};
+  let box: Rectangle | undefined = undefined;
```

Comment:
```suggestion
  let box: Rectangle;
```
I think this is actually enough 🤔

Reply:
It is actually complaining because it's not assigned to in all code paths